src/vue-pdf-embed.vue

```diff
@@ -13,6 +20,14 @@ pdf.GlobalWorkerOptions.workerPort = new PdfWorker()
 export default {
   name: 'VuePdfEmbed',
   props: {
+    /**
+     * Id of the document to display 
+     * @values String
+     */
+    documentId: {
```

Comment:
In addition to the above

```suggestion
    id: {
```